skin.css

```diff
@@ -2,12 +2,65 @@
     background-color: #ddd;
 }
 
-button.btn {
+.btn {
+  width:170px
   border-radius: 3px;
-  border-style: solid;
-  border-width: 1px;
+  background-color:white;
   font-weight: normal;
-  line-height: 1.15;
-  text-align: center;
-  vertical-align: middle;
+  font-family:sans-serif;
+  padding: 15px;
+  border: 1px; solid lightgrey
+
+
+}
+
+
+  .btn--default{
+   border-style:lightgrey
+    
+       } 
+.btn--primary{
+  background-color:blue;
+  color:white;
+  }
+  .btn--secondary{
+    background-color:gainsboro;
+    color:grey;
+    
+  }
+  .btn--large{
+    width:250px;
+    padding:20px;
+    color:blue;
+  }
+  .btn--extra-large{
+    width:350px;
+    padding:20px;
+    background-color: white;
+    color: blue;
+    font-size:20px;
+
+  }
+  nav.ul{
+    float: left;
+    
+  }
+  ul li{
+    float: left;
+    padding:20px;
+    margin: 20px;
+    width: 80px;
+    list-style-type: none;
+    border: 1px dotted grey;
+    text-align: center;
+}
+  li a{

```

Comment:
Same thing as above - in order to get this to apply only to `nav`, you'll need to use `nav ul li a`. This is where SASS comes to help!

Without SASS
```
.nav { ... }
.nav ul { ... }
.nav ul li { ... }
.nav ul li:hover { ... }
.nav ul li:first-child { ... }
```

With SASS
```
.nav {
  ...
  
  ul {
    ...
    
    li {
      ...
      
      &:hover { ... }
      &:first-child { ... }
    }
  }
}
```